.github/workflows/generate-data.yaml

```diff
@@ -22,15 +22,16 @@ on:
         required: true
 
 jobs:
-  build-data:
-    name: "Generate Data"
+  check:
+    name: "Check Bot Status"
     runs-on: ubuntu-latest
-    continue-on-error: true
-    strategy:
-      matrix:
-        site: ${{ fromJSON(inputs.repos) }}
+    defaults:
+      run:
+        shell: bash
+    outputs:
+      exists: ${{ steps.bot.outputs.bot }}
     steps:
-      - id: is-bot
+      - id: bot
```

Comment:
There was a check that I had duplicated in every job that checked if we were running a bot. I moved it up to its own job so that it has the output variable of `exists`, which allows us to generate a token or not.